src/kagglehub/gcs_upload.py

```diff
@@ -49,11 +51,13 @@ def _upload_blob(file_path: str, model_type: str):
     file_path: The path to the file to be uploaded.
     model_type : The type of the model associated with the file.
     """
+    file_size = os.path.getsize(file_path)
     data = {
         "type": model_type,
         "name": os.path.basename(file_path),
-        "contentLength": os.path.getsize(file_path),
+        "contentLength": file_size,
         "lastModifiedEpochSeconds": int(os.path.getmtime(file_path)),
+        "resumable": True,
```

Comment:
Let's remove this and add it in the next PR adding support for resumable uploads.